resources/troubleshooter/data.json

```
@@ -407,6 +407,21 @@
                 }
             ]
         },
+        "servo-plod": {
+            "question": "Is there a purple LED on the Servo Board?",
```

```suggestion
            "question": "Is there a pink LED on the Servo Board?",
```

I'd say it's more of a pink light, the red LED is much brighter than the blue